src/ovi/Data/Configuration.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Upmind\ProvisionProviders\SharedHosting\ovi\Data;
+
+use Upmind\ProvisionBase\Provider\DataSet\DataSet;
+use Upmind\ProvisionBase\Provider\DataSet\Rules;
+
+/**
+ * Example API credentials.
+ *
+ * @property-read string $hostname Plesk ovi hostname
```

Comment:
For IDE auto-completion please add [PHPDoc `@property-read`](https://docs.phpdoc.org/2.9/references/phpdoc/tags/property-read.html) blocks for each Configuration value